packages/psss/src/api/queries/useUpcomingReport.js

```diff
@@ -16,7 +16,7 @@ export const useUpcomingReport = countryCode => {
   const query = useQuery(
     [endpoint, lastPeriod],
     () => get(endpoint, { params: { startWeek: lastPeriod, endWeek: lastPeriod } }),
-    { staleTime: 60 * 1000 },
+    { staleTime: 60 * 1000 * 5 },
```

Comment:
The stale time is how long the query is cached for (except when explicitly cleared). 5 minutes seems fine. I've made a note to check with Edwin when he's back too.